src/lucky_web/date_helpers.cr

```diff
@@ -0,0 +1,47 @@
+module LuckyWeb::DateHelpers
+  def distance_of_time_in_words(from : Time, to : Time)
+    minutes = (to - from).minutes
+    seconds = (to - from).seconds
+    hours = (to - from).hours
+    days = (to - from).days
+
+    return distance_in_days(days) if days != 0
```

Comment:
Instead of returning a string, can you call `text` in this method and in `time_ago_in_words`? That way in pages you only need to do:

```crystal
time_ago_in_words Time.new - 1.minute

# instead of
text time_ago_in_words(Time.new - 1.minute)`
```

This would mean you can't use these helpers outside of views, but that is ok for now. We can figure out how to extract them later on. I don't think it will be too tricky

Reply:
No problem I figure it out.